### DISCUSSION TITLE
[Q] Remove a vertex of a Polyline by Ctrl+Click on Edit mode ?

### DISCUSSION THREAD

**Discussion by lndrtrbn** [geoman-io/leaflet-geoman#846] [Questions & Help]
I'm trying to remove a specific vertex on Edit mode by Ctrl-Click on a vertex. I can catch the click event on a vertex and detect if Ctrl is pressed but I'm stuck trying to remove this vertex.

Example : https://jsfiddle.net/rq65jz0e/2/
1. Create a 3 vertexes line
2. Try to remove the middle vertex with Ctrl-Click

For now my attempts lead to removing the entire Line or just the Marker of the vertex.
Am I wrong in the way I implement it ?

**Comment by Falke-Design** [comment]
Use this code:
```
  e.layer.on('pm:vertexclick', function(x) {
    if (x.markerEvent.originalEvent.ctrlKey) {
        x.layer.pm._removeMarker(x.markerEvent);
    }
  })
```

I think this can be also an interessting Feature Request #817

**Comment by lndrtrbn** [comment]
Thanks !
